app/views/challenges/12in23.html.haml

```diff
@@ -0,0 +1,35 @@
+#challenge-12in23-page
+  %section.top-section
+    .lg-container.c-shapes
+      %h1.text-h1.mb-16 The #12in23 Challenge
+      %p
+        Below you'll see a list of all the languages you've used during 2023.
+        Aim to complete 12 languages by the end of the year.
+
+        Join in the conversation by using the
+        = link_to "#12in23 tag", "https://forum.exercism.org/tag/12in23"
+        on our forums or the
+        %strong.font-semibold #12in23
+        hashtag around social media!
+      %p.cta
+        Complete 5 exercises (not including "Hello World") in a track to tick off that language!
```

Comment:
The track counts query does not check for completed exercises, but only for solutions with a submitted iteration. Is the query wrong?

Reply:
Updated the text.